templates/physical-architecture/physical-link.html.j2

```diff
@@ -11,9 +11,11 @@
 <p>A part of {{ linked_name(object.parent) | safe }}</p>
 {% endif %}
 
-<p>Connects {{ linked_name(object.ends[0].owner) | safe}} via {{ linked_name(object.ends[0]) | safe }}  port and {{ linked_name(object.ends[1].owner) | safe }} via {{ linked_name(object.ends[1]) | safe }} port.</p>
+{{ object.context_diagram.as_svg | safe }}
```

Comment:
if `context_diagram` is not defined, this will throw an error. Are we expected for this object to always have a context diagram? if not i would suggest we add a check here

Reply:
capellambse-context-diagrams is listed as dependency in pyproject.toml, so it's safe to assume it will be there.

Reply:
I am not sure if this would even be a problem if context diagrams would be missing in the used virtualenv. Wouldn't it then be `undefined` and that's it?